src/views/ProfileInjectedTypeform.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Typeform from "../components/Typeform/typeform";
+import { useRecoilValue } from "recoil";
+import { auth_gsuite } from "../api/state";
+
+interface message {
+  typeform_id: string;
+}
+const ProfileInjectedTypeform = (props: message) => {
+  const auth_status = useRecoilValue(auth_gsuite);
+  const email =
+    auth_status.decoded_jwt === undefined
+      ? undefined
+      : auth_status.decoded_jwt["email"];
+  const name = email!.split("@")[0];
+  const firstName =
+    name.split(".")[0].charAt(0).toUpperCase() + name.split(".")[0].slice(1);
```

Comment:
Can we make a function called `CapitalizeFirstLetter()` just to make it clear what's happening here?